server/blob/blob-api/src/test/java/org/apache/james/blob/api/ReadSaveBlobStoreDAOContract.java

```diff
@@ -236,7 +236,7 @@ default void saveBytesShouldBeIdempotent(String description, byte[] bytes) {
     @MethodSource("blobs")
     default void saveByteSourceShouldBeIdempotent(String description, byte[] bytes) {
         BlobStoreDAO store = testee();
-        Mono.from(store.save(TEST_BUCKET_NAME, TEST_BLOB_ID, ByteSource.wrap(bytes))).block();
+        Mono.from(store.save(TEST_BUCKET_NAME, TEST_BLOB_ID, TWELVE_MEGABYTES)).block();
```

Comment:
Not sure if it's acceptable or not....

It seems for some reason first save fails if it's with the ByteSource or InputStream with a big blob, but not with the direct bytes. Also if before the two ByteSource big blob saves you do a save on a smaller blob, then it's green.

It's rather odd and I don't really understand what's going on there, if it's related to MinIO warming up or bucket data init... Will try to dig more but... looks rather weird to me.

If someone else with more object storage or MinIO knowledge wants to have a look as well, feel free :)